core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -757,10 +796,11 @@ public void extractAttachment(Attachment attachment, SaxElementBuilder attachmen
 
 	@Override
 	public String getPhysicalDestinationName() {
-		if (exchangeService != null) {
-			return "url [" + (exchangeService == null ? "" : exchangeService.getUrl()) + "] mailAddress [" + (getMailAddress() == null ? "" : getMailAddress()) + "]";
+		String result = super.getPhysicalDestinationName();
+		if (globalExchangeService != null) {
+			result = Misc.concatStrings("url [" + (globalExchangeService == null ? "" : globalExchangeService.getUrl()) + "] mailAddress [" + (getMailAddress() == null ? "" : getMailAddress()) + "]", " ", result);
```

Comment:
Aangezien de `PhysicalDestinationName` redelijk vaak opgehaald wordt is het misschien handig de url alleen weer te geven als deze daadwerkelijk opgegeven is, en anders 'using autodiscovery' te gebruiken.

Reply:
Ik denk dat als je autodiscovery gebruikt je ook de url wilt zien, zodat je die vervolgens kan opgeven. Autodiscovery is erg traag.